_sass/components/popovers/popover-search.scss

```diff
@@ -0,0 +1,72 @@
+.popover-search {
+  border: 0;
+  bottom: 0;
+  height: auto;
+  overflow-x: hidden;
+  overflow-y: scroll;
+  position: relative;
+
+  &:not(.lunr-hidden) {
+    background: #fff;
+    max-height: 349px;
```

Comment:
I think that max-height, width, background, and transition are all styles that don't belong in this selector (as well as the media query).